.cirrus.yml

```diff
@@ -28,6 +28,7 @@ task:
     matrix:
       # A stable 13.0 image likely won't be available before early 2021
       # image_family: freebsd-13-0-snap
+      image_family: freebsd-12-2
       image_family: freebsd-12-1
```

Comment:
can we just use the latest 12x instead of both, is there a disadvantage to that

Reply:
I would ask the opposite. Is there a good reason to not to? Many people are still using 12.1 and AFAIK it's not dead. See freebsd.org, it is still listed on Downloads there

Reply:
I suppose the question is if the difference between the two versions 12.1 and 12.2 is big enough to warrant two different jobs. We don't run different jobs on other systems for different release minor version changes.

Reply:
After investigating the changelog I came to the conclusion that it's probably better to keep both versions. For example make(1) was updated which is relevant for our build system.